docs/modelling/index.rst

```diff
@@ -0,0 +1,8 @@
+Modelling
```

Comment:
Typically spelled with one L.  This is very nit-picky, but I mention it to raise a more general question in...should our docs use *American* English?